File: _BHCVRP_Python_Version/BHCVRP_Python_Version/cujae_inf_citi_om/exceptions/CostException.py
class CostException(Exception):
    def __init__(self, cost=None):
        if cost is not None:
            self.request = cost
        
    def __init__(self, message=None):
        if message is not None:
            super().__init__(message)

    def get_cost(self):
        return getattr(self, 'cost', None)

File: BHCVRP_Java_Version/src/cujae/inf/citi/om/exceptions/CostException.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package cujae.inf.citi.om.exceptions;

/**
 *
 * @author nanda
 */

public class CostException extends Exception{
    private float cost;
    
    public CostException(float cost){
        this.cost = cost;
    
    }
    
    public float getCost(){
        return cost;
    }
    
    public CostException(String message){   
        super(message);
    }
    
}
